hack/lib/util.sh

```diff
@@ -124,60 +155,51 @@ kube::util::wait-for-jobs() {
   return ${fail}
 }
 
-# Takes a binary to run $1 and then copies the results to $2.
-# If the generated and original files are the same after filtering lines
-# that match $3, copy is skipped.
-kube::util::gen-doc() {
-  local cmd="$1"
-  local base_dest="$(kube::util::realpath $2)"
-  local relative_doc_dest="$(echo $3 | sed 's/\/$//')"
-  local dest="${base_dest}/${relative_doc_dest}"
-  local skipprefix="${4:-}"
-
-  # We do this in a tmpdir in case the dest has other non-autogenned files
-  # We don't want to include them in the list of gen'd files
-  local tmpdir="${KUBE_ROOT}/doc_tmp"
-  mkdir -p "${tmpdir}"
-  # generate the new files
-  ${cmd} "${tmpdir}"
+# $1 is the directory to put the generated documents in
+kube::util::gen-docs() {
```

Comment:
End goal is like https://github.com/kubernetes/kubernetes/pull/13018/files#diff-ef2873ebed657da627ed30f61dea2964R158 where it does it for everything...